core/src/main/scala/org/apache/pekko/persistence/r2dbc/query/scaladsl/mysql/MySQLQueryDao.scala

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
this looks brand new (no code copied from an existing source file) - therefore, could you use the standard license header (like the create sql)?